README.md

```diff
@@ -103,7 +103,12 @@ You may also choose to hide the conference call icon in either channels or priva
 }
 ```
 ### Running your own conference server
-Running your own conference server allows you to secure your conference rooms. We make use of the Jitsi Meet server to handle the conference calls, the installation steps can be found here: https://github.com/jitsi/jitsi-meet/blob/master/doc/quick-install.md
+Running your own conference server allows you to secure your conference rooms. We make use of the Jitsi Meet server to handle the conference calls, the installation steps can be found here: https://jitsi.github.io/handbook/docs/devops-guide/devops-guide-quickstart
+
+If docker is your preferred method then see here: https://jitsi.github.io/handbook/docs/devops-guide/devops-guide-docker
```

Comment:
I would take this a step further and say that we recommend the docker installation since it's a lot easier to install and maintain. WHich makes our supporting it much easier